1732Sprint/4-Xander/CH1-Demo/pets/PetGen.java

```diff
@@ -0,0 +1,34 @@
+package pets;
+
+import java.util.Random;
+
+public class PetGen {
+  String nameArray[] = {"Max","Charlie","Buddy","Rocky","Buster","Bear","Winston","Spike","Brutus"};
+  String speciesArray[] = {"dog","cat","gecko","girrafe","axolotl","ferret","pet rock"};
+  String species;
+  String name;
+
+  public PetGen() {
+    this.setRandomName();
+    this.setRandomSpecies();
+  }
+
+  public void setRandomName() {
+    Random r = new Random();
+    this.name = this.nameArray[r.nextInt(this.nameArray.length) + 0];
+  }
+
+  public String getName() {
+    return this.name;
+  }
+
+  public void setRandomSpecies() {
```

Comment:
No, but since chapter 1 did not go into access modifiers I thought it would be easiest to leave it public.